src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+
+        String action = null;
+        Scanner input = new Scanner(System.in);
+        Task[] allTasks = new Task[100];
+        int numTasks = 0;
+        int taskNum = 0;
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
Maybe can change to constant (final static) since logo is not expected to change